command/format/diagnostic.go

```diff
@@ -180,12 +177,6 @@ func Diagnostic(diag tfdiags.Diagnostic, sources map[string][]byte, color *color
 	return buf.String()
 }
 
-// sourceCodeContextStr attempts to find a user-friendly description of
```

Comment:
Is this deleted because it's referring to a function that's not here anymore?

Reply:
Yes. The function was renamed and the behavior changed in an earlier merge, so I just removed the comment. I would've updated the comment but I wasn't sure how to properly describe the new behavior :(

Reply:
This does indeed seem kinda weird... from the description of `sourceCodeContextStr` it seems like it was the former home of what is now [`hcled.ContextString`](https://godoc.org/github.com/hashicorp/hcl2/hcled#ContextString), the function that figures out what top-level block a given offset is inside so we can include that in the error messages. It looks like this `parseRange` function was what remained after we factored out the main work into that `hcled.ContextString` function; the work now is just to go from raw source code and a range into suitable arguments to pass to `hcled.ContextString`. `parseRange` feels like an odd name for that, but I don't have a better one... :thinking: